HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java

```diff
@@ -74,9 +110,28 @@ public RedirectView createRecruiter(@ModelAttribute RecruiterDTO recruiter, Http
         }
         return redirectView;
     }
+    
+    @RequestMapping(value = "/handle-Applicant", method = RequestMethod.POST)
+    public RedirectView createApplicant(@ModelAttribute ApplicantDTO applicant, HttpServletRequest request) {
+        System.out.println(applicant);
+        RedirectView redirectView = new RedirectView();
```

Comment:
Remove Print Statements.. before pushing clean up your code to remove unwanted lines

Reply:
ok